src/Zendesk.hs

```diff
@@ -144,6 +145,69 @@ runZendeskMain = do
             printWarning
             tickets <- runApp (listTickets Assigned) cfg
             printTicketCountMessage tickets (cfgEmail cfg)
+        ShowStats2 -> do
+            putTextLn $ "Testing - Statistics associated with: "
+               <> cfgEmail cfg
+            tickets <- runApp (listTickets Assigned) cfg
+            statsExample
+            -- How many tickets are open
+            putTextLn $ "There are " <> show (length $ filter ((== "open") . ticketStatus) tickets) <> " open tickets."
```

Comment:
You can put things like `length $ filter ((== "open") . ticketStatus) tickets` in separate functions to increase readability.